libs/core/langchain_core/output_parsers/openai_tools.py

```diff
@@ -252,7 +254,7 @@ def parse_result(self, result: List[Generation], *, partial: bool = False) -> An
 class PydanticToolsParser(JsonOutputToolsParser):
     """Parse tools from OpenAI response."""
 
-    tools: List[TypeBaseModel]
+    tools: Annotated[List[TypeBaseModel], SkipValidation()]
```

Comment:
are we getting rid of TypeBaseModel?

Reply:
We could try to support .v1 models for a bit of time to avoid more breaking changes for users, but can also drop now